internal/iso_packaging/iso_packaging.go

```diff
@@ -0,0 +1,241 @@
+package iso_packaging
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/project-flotta/osbuild-operator/api/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+const (
+	templateCommand string = "isopackage %s -ks %s --upload-target"
+	kickstart       string = "init.ks"
+)
+
+var (
+	jobTTLafterFinish int32 = 100   // Job will be deleted after finished with this
```

Comment:
Sorry for nitpicking here. It should be a capital A: `jobTTLAfterFinish`

Reply:
:+1: